node/tessel-export.js

```diff
@@ -578,7 +585,8 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
 Tessel.Pin.prototype.on = Tessel.Pin.prototype.addListener;
 
 Tessel.Pin.prototype._setInterruptMode = function(mode) {
-  this.interruptMode = mode;
+  // rise and fall events will be emitted by change event
+  (mode === 'rise' || mode === 'fall') ? this.interruptMode = 'change' : this.interruptMode = mode;
```

Comment:
It is confusing when the assignment isn't on the left.

Using what @LinusU provided would be fine and pass JSHint. One nit I would have would be that the outer set of parenthesis are not needed.